.travis.yml

```diff
@@ -35,9 +34,36 @@ before_install:
 script:
 - yarn test
 - yarn run build
-after_success:
-- test $TRAVIS_BRANCH = "master" && pngquant build/screenshots/* -f --ext ".png"
-- test $TRAVIS_BRANCH = "master" && test "$TRAVIS_PULL_REQUEST" = "false" && env DEPLOY_BRANCH=build yarn run deploy
-- test $TRAVIS_BRANCH = "feature/one-tile-per-provider" && test "$TRAVIS_PULL_REQUEST" = "false" && env DEPLOY_BRANCH=build_one_tile_per_provider yarn run deploy
-- test $TRAVIS_BRANCH = "orange" && test "$TRAVIS_PULL_REQUEST" = "false" && env DEPLOY_BRANCH=build_orange yarn run deploy
-- test $TRAVIS_BRANCH = "prod" && test "$TRAVIS_PULL_REQUEST" = "false" && env DEPLOY_BRANCH=latest yarn run deploy
+before_deploy:
+- yarn add cozy-app-publish
+deploy:
+  - provider: script
+    repo: cozy/cozy-collect
+    skip-cleanup: true
+    script: DEPLOY_BRANCH=build && yarn deploy && yarn cozyPublish
+    on:
+      branch: master
+  - provider: script
+    repo: cozy/cozy-collect
+    skip-cleanup: true
+    script: DEPLOY_BRANCH=latest && yarn cozyPublish
+    on:
+      tags: true
```

Comment:
This way we can use Github Release even on master branch if we want, it won't change the publishing and use the correct latest branch for that. But we will have to wait the latests `prod` build finished and deployed to be sure to have the last build on `latest`.